docs/eventing/sources/sinkbinding/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "SinkBinding"
+weight: 01
+type: "docs"
+aliases:
+  - /docs/eventing/samples/sinkbinding/index
+  - /docs/eventing/samples/sinkbinding/README
+  - /docs/eventing/sources/sinkbinding.md
+---
+
+![API version v1](https://img.shields.io/badge/API_Version-v1-red?style=flat-square)
+
+The `SinkBinding` custom object supports decoupling event production from
+delivery addressing.
+
+You can use a SinkBinding to direct a subject to an event sink. The _subject_
+is a Kubernetes resource that embeds a `PodSpec` template and produces events.
+The _event sink_ is an addressable Kubernetes object that can receive events.
+
+To create new subjects you can use any of the compute objects that Kubernetes
+makes available such as:
+
+- `Deployment`
+- `Job`
+- `DaemonSet`
+- `StatefulSet`
+
+You can also create new subjects that use Knative abstractions, such as
+`Service` or `Configuration` objects.
+
+The SinkBinding injects environment variables into the `PodTemplateSpec` of the
```

Comment:
```suggestion
Sink binding injects environment variables into the `PodTemplateSpec` of the
```

Reply:
Only refer to SinkBinding if we mean the literal object / custom resource here, it seems more conceptual though? Just a suggestion

Reply:
I think this is referring to what the literal object does, right? I'll update to "SinkBinding object".